app/src/main/java/com/x8bit/bitwarden/data/platform/datasource/disk/SettingsDiskSourceImpl.kt

```diff
@@ -395,4 +396,15 @@ class SettingsDiskSourceImpl(
         getBoolean(
             key = HAS_USER_LOGGED_IN_OR_CREATED_AN_ACCOUNT_KEY.appendIdentifier(userId),
         ) == true
+
+    override fun getShowAutoFillSettingBadge(userId: String): Boolean =
+        getBoolean(
+            key = SHOW_AUTOFILL_SETTING_BADGE.appendIdentifier(userId),
+        ) == true
+
+    override fun storeShowAutoFillSettingBadge(userId: String, showBadge: Boolean) =
```

Comment:
This needs to be nulled out `clearData`, right?

Reply:
I am not sure, cause technically they will not through onboarding again on the device unless the data is cleared manually. I originally had this but then cause of the above figured to leave it out? Thoughts?

Reply:
We should probably just verify that when a user is logged out, they do not see the onboarding flow again.

I would think they need to re-setup their biometric or pin unlock still. If they do not need to see it again, could we not it in `clearData`. There should already be a comment section in there for it.

Reply:
Just to follow up on this, it appears we do want this to just be a one time thing per user like the onboarding